students/ABartles/Exercise6/test_mailroom.py

```diff
@@ -0,0 +1,67 @@
+from mailroom import *
+import os
+
+'''
+Items to Test:
+    sum_gifts
+    count_gifts
+    sorted donors
+    list_donors
+    *thanks letter
+    *send_thanks_all
+
+Items with user interaction: (can pytest test/simulate user imput?)
+    rec_donation
+    send_thanks_one
+    menu
+
+Not sure how to test w/ pytest:
+    create_report
+    run_func
+    __name__ == __main__
+'''
+
+def test_1():
+    assert sum_gifts("Anita Bath") == 350
```

Comment:
your test names should be helpful, test_1 is too vague; a good name is `test_sum_gifts`